site/source/_posts/2017/11/17-Dojo-2-Beta-4/Dojo-2-Beta-4.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Dojo 2 Beta 4
+date: 2017-11-17 18:00:00
+authorId: kitsonk
+featured_image: featured.png
+---
+
+The Beta 4 Release of Dojo 2 puts us on a clear path to releasing Dojo 2.  It has several significant changes and updates.
+
+{% asset_img featured.png feature-full %}
+
+<!-- more -->
+
+## Getting Beta 4
+
+Beta 4 is currently tagged as `latest` in npm, so all just updating to the latest version will provide you with Beta 4.  We also have a release tag of `beta4` in npm if you want to focus just on using Beta 4.
```

Comment:
so all just updating to the latest version will provide you with Beta 4 -> so simply updating to the latest version will provide you with the Beta 4 release.